keyboards/westfoxtrot/prophet/prophet.c

```diff
@@ -0,0 +1,19 @@
+#include "prophet.h"
+
+void keyboard_pre_init_kb (void) {
+  setPinOutput(B12);
+  setPinOutput(B13);
+}
+
+bool led_update_kb(led_t led_state) {
+  bool res = led_update_user(led_state);
+  if(res) {
+    writePin(B13, led_state.caps_lock);
+  }
+  return res;
+}
+
+__attribute__((weak)) layer_state_t layer_state_set_user(layer_state_t state) {
+  writePin(B12, (state & (1UL << 1)));
```

Comment:
You could also do this: 
```suggestion
  writePin(B12, layer_state_cmd(state, 1));
```

Reply:
`keyboards/westfoxtrot/prophet/prophet.c:17:17: error: implicit declaration of function 'layer_state_cmd'; did you mean 'layer_state_cmp'? [-Werror=implicit-function-declaration]`

I get this error with this change

Reply:
it should be `layer_state_cmp` as the error suggests.